pytorch_binding/warpctc_pytorch/__init__.py

```diff
@@ -52,6 +52,46 @@ def forward(ctx, acts, labels, act_lens, label_lens, size_average=False,
     def backward(ctx, grad_output):
         return ctx.grads * grad_output.to(ctx.grads.device), None, None, None, None, None, None
 
+class _CTC_irreduce(Function):
```

Comment:
Would you add `reduce` option and reduction process to `_CTC.forward`, because averaging process is a kind of reduction?

Please add conditions of reduction  process to 
https://github.com/jnishi/warp-ctc/blob/pytorch_bindings/pytorch_binding/warpctc_pytorch/__init__.py#L36-L46 like below.

```python
   if reduce:
       if length_average:
            ...foo...
       elif size_average:
           ...bar...
   else:
      ...baz...
```